internal/services/eventhub/eventhub_namespace_schema_registry_resource.go

```diff
@@ -0,0 +1,212 @@
+package eventhub
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/eventhub/2021-11-01/namespaces"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/eventhub/2021-11-01/schemaregistry"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/eventhub/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+)
+
+func resourceEventHubNamespaceSchemaRegistry() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceEventHubNamespaceSchemaRegistryCreateUpdate,
+		Read:   resourceEventHubNamespaceSchemaRegistryRead,
+		// Update: resourceEventHubNamespaceSchemaRegistryCreateUpdate,
+		Delete: resourceEventHubNamespaceSchemaRegistryDelete,
+
+		Importer: pluginsdk.ImporterValidatingResourceId(func(id string) error {
+			_, err := schemaregistry.ParseSchemaGroupID(id)
+			return err
+		}),
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			// Update: pluginsdk.DefaultTimeout(30 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*pluginsdk.Schema{
+			"name": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.ValidateSchemaGroupName(),
+			},
+
+			"namespace_id": {
+				Type:         pluginsdk.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: namespaces.ValidateNamespaceID,
+			},
+
+			"schema_compatibility_type": {
```

Comment:
does type make any sense here?
```suggestion
			"schema_compatibility": {
```

Reply:
Yes, resource is functioning just fine without this property. I'll modify the naming

Reply:
The resource is stable without this `groupProperty`. It's not a required one when creating the schema group in the portal:
![image](https://user-images.githubusercontent.com/92154856/183789895-9f7f7ba7-fdfd-48ff-a838-2f812807a432.png)

based on my understanding, it should have the same funtion as `Tag` in the resource: user to speficy the key and the corresponding value.